locations/spiders/delhaize_be.py

```diff
@@ -0,0 +1,15 @@
+from scrapy.spiders import SitemapSpider
+
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class DelhaizeBESpider(SitemapSpider, StructuredDataSpider):
+    name = "delhaize_be"
+    item_attributes = {
+        "brand": "Delhaize",
+        "brand_wikidata": "Q1184173",
+        "country": "BE",
+    }
+    sitemap_urls = ["https://stores.delhaize.be/sitemap.xml"]
+    sitemap_rules = [(".*/fr/.*", "parse_sd")]
```

Comment:
If it's regex, we should prefix the string with r `r".*/fr/.*"`
```suggestion
    sitemap_rules = [(r".*/fr/.*", "parse_sd")]
```
However I imagine we can do this without regex, if so:
```suggestion
    sitemap_rules = [("/fr/", "parse_sd")]
```

No preference from me